packages/grz-common/src/grz_common/models/s3.py

```diff
@@ -53,6 +53,21 @@ class S3Options(IgnoringBaseModel):
     The proxy URL for S3 operations (optional).
     """
 
+    proxy_ca_bundle: str | None = None
+    """
+    The path to a custom certificate bundle to use when establishing SSL/TLS connections with proxy (optional).
+    """
+
+    proxy_client_cert: str | None = None
+    """
+    The path to a certificate for proxy TLS client authentication  (optional).
+    """
+
+    proxy_use_forwarding_for_https: bool = False
+    """
+    For HTTPS proxies, forward your requests to HTTPS destinations with an absolute URI. We strongly recommend you only use this option with trusted or corporate proxies.
+    """
```

Comment:
Could you group these under a `proxy_config` key to mimic the boto3 layout?

e.g.

```yaml
proxy_config:
  proxy_use_forwarding_for_https: yes
```